lib/Sema/CSApply.cpp

```diff
@@ -1087,7 +1087,7 @@ namespace {
                                 locator);
 
         auto *const calleeParamDecl = calleeParamList->get(idx);
-        if (calleeParamDecl->hasAttachedPropertyWrapper()) {
+        if (calleeParamDecl->hasAttachedPropertyWrapper() && calleeParamDecl->hasExternalPropertyWrapper()) {
```

Comment:
@hborla `calleeParamDecl->hasAttachedPropertyWrapper()` could be removed from this check, right?

Reply:
This is a much stricter check now too.

Reply:
That's right, `hasAttachedPropertyWrapper()` should be redundant. The check is stricter, but it's also correct; only external property wrappers have an impact on arguments at the call-site (whereas internal property wrappers on parameters just create a local wrapped variable inside the function's implementation).

Reply:
Correct, I will adjust it.

Reply:
I have pushed changes.